_observing-your-data/trace/distributed-tracing.md

```diff
@@ -0,0 +1,162 @@
+
+- **Instrumentation:** Instrumenting your application code with OpenTelemetry SDKs.
+- **Propagation:** Injecting trace context into requests as they propagate through your system.
+- **Collection:** Collecting trace data from your application.
+- **Processing:** Aggregating trace data from multiple sources and enriching it with additional metadata.
+- **Exporting:** Sending trace data to a backend for storage and analysis. 
+
+OpenSearch is often chosen as the sink for storing trace data.
+
+## Trace analytics
+
+OpenSearch provides a `trace-analytics` plugin for visualizing trace data in real time. The plugin includes pre-built dashboards for analyzing trace data, such as service maps, latency histograms, and error rates. With OpenSearch's distributed tracing pipeline, you can quickly identify bottlenecks and errors in your applications. See the [Trace analytics]({{site.url}}{{site.baseurl}}/observing-your-data/trace/index/) documentation for more information. 
+
+## Get started
+
+The distributed tracing feature is experimental as of OpenSearch 2.10. To begin using the distributed tracing feature, you need to first enable it using `opensearch.experimental.feature.telemetry.enabled` feature flag, and subsequently activate the tracer, using the dynamic setting `telemetry.tracer.enabled`. It's important to exercise caution when enabling this feature, as it can consume system resources. Detailed information on enabling and configuring distributed tracing, including on-demand troubleshooting and request sampling, is described in the following sections.
+
+### Enable on a node using a tarball install
+
+The flag is toggled using a new Java Virtual Machine (JVM) parameter that is set either in `OPENSEARCH_JAVA_OPTS` or in `config/jvm.options`.
+
+#### Option 1: Enable the experimental feature flag in the `opensearch.yml` file
+
+1. Change to the top directory of your OpenSearch installation.
+
+```bash
+cd \path\to\opensearch
+```
+
+2. Open your OpenSearch configuration folder, then open the `opensearch.yml` file with a text editor.
+3. Add the following line:
+
+```bash
+opensearch.experimental.feature.telemetry.enabled=true
+```
+{% include copy.html %}
+
+4. Save your changes and close the file.
+
+#### Option 2: Modify jvm.options
+
+Add the following lines to `config/jvm.options` before starting the OpenSearch process to enable the feature and its dependency:
+
+```bash
+-Dopensearch.experimental.feature.telemetry.enabled=true
+```
+{% include copy.html %}
+
+Run OpenSearch
+
+```bash
+./bin/opensearch
+```
+{% include copy.html %}
+
+#### Option 3: Enable from an environment variable
+
+As an alternative to directly modifying `config/jvm.options`, you can define the properties by using an environment variable. This can be done in a single command when you start OpenSearch or by defining the variable with export.
```

Comment:
Are there words missing at the end of this sentence? It seems like an abrupt stop.

Reply:
Revised